pom.xml

```diff
@@ -106,7 +106,13 @@
         <dependency>
             <groupId>org.apache.oltu.oauth2</groupId>
             <artifactId>org.apache.oltu.oauth2.client</artifactId>
-            <version>0.31</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.codehaus.jettison</groupId>
+                    <artifactId>jettison</artifactId>
+                </exclusion>
+            </exclusions>
+            <version>${org.apache.oltu.oauth2.client-version}</version>
```

Comment:
Shall we change the order? the version comes first and then the exclusions? WDYT?

Reply:
Done